lua/modules/configs/completion/formatting.lua

```diff
@@ -156,18 +157,19 @@ function M.format(opts)
 				{ title = "LSP Formatter Warning" }
 			)
 			return
-		elseif
-			format_modifications_only
-			and require("lsp-format-modifications").format_modifications(client, bufnr).success
-		then
-			if format_notify then
-				vim.notify(
-					string.format("[LSP] Format changed lines successfully with %s!", client.name),
-					vim.log.levels.INFO,
-					{ title = "LSP Range Format Success" }
-				)
+		end
+
+		if format_modifications_only then
```

Comment:
I'm a bit confused as to why this branch was separated from the previous `if` statement. Shouldn't formatting stop when the current filetype is in the blocklist?

Reply:
it will early `return` in the branch of `block_list[vim.bo.filetype] == true`.

Reply:
oops you're right. Sorry for the oversight here! btw imo combining the following two `if` statements would improve readability, as they are essentially the prerequisites for `format_notify`:
```suggestion
		if
			format_modifications_only
			and require("lsp-format-modifications").format_modifications(client, bufnr).success
		then
```
Any thoughts on this?